README.md

```diff
@@ -4,4 +4,4 @@ Connect with people based on your music tastes.
 ## Description
 If you'd like to attend a show of a band that none of your friends is into, if you feel like meeting a special one who also **loves** that track, or if you just want to get to know new people that enjoy the same bands as you, *music-matcher* is here to help you out.
 
-From your Spotify listening story, *music-matcher* will suggest people how would also be eager to meet you.
\ No newline at end of file
+From your Spotify listening story, *music-matcher* will suggest people who will also be eager to meet you.
```

Comment:
:clap: for fixing before merging.

Reply:
Thanks!! I'll keep trying my best :muscle: